awesomemotive/one-click-demo-import
# Fix PHPUnit

## Description

This PR includes the following:
1. Add PHPUnit Polyfill in `composer` dev-dependencies. And load them in the test's bootstrap file. This is required to run the `phpunit` test suite.
2. Replace the URL `http://invalid-url.com` with `http://invalid-url.test` in the test. I believe that the former was used when the `http://invalid-url.com` was indeed still invalid. But at the moment it's no longer invalid and it's loading a webpage. I used `.test` since it's guaranteed to never be registered on the internet. [Reference](https://en.wikipedia.org/wiki/.test).

## Testing procedure

1. Run `composer install` in the repo root.
2. Run `phpunit`. See [WIKI](https://github.com/awesomemotive/one-click-demo-import/wiki/How-to-run-PHPunit-tests) on how.

## Screenshot

https://a.supportally.com/i/Nf5LTO
